### ISSUE TITLE
Timm version

### ISSUE THREAD

**Issue by christiaantheunisse** [noahzn/Lite-Mono#8]
Hi @noahzn,

I am trying to train the model, so I followed the procedure from monodepth2. I created the conda environment with the right packages and versions, but when I run the training, it get an error saying that `timm` is missing. I tried to install a few different versions, but it keeps throwing errors. I also tried it with pytorch 1.0, since that is also possible according to the monodepth2 page, but that doesn't work neither.

Can you tell me which `timm` package I need and if I should use the same package versions as monodepth2.

Thanks in advance!

**Comment by christiaantheunisse** [comment]
Hi @noahzn ,
Thank you for your reply! I got it working now.
I am setting up a google VM, so I installed all the newest version of the packages. However, I got some error about 30 seconds after I started the training about the dataloader, because I used pytorch `2.0` (at least that is what I found after some research on stackoverflow). Unfortunately, I cannot remember the exact error.
So I tried to install the version mentioned on the monodepth2 github, but these were to old to be compatible with timm. With the versions you provided, I got it working. I guess its just a rookie mistake, so thanks for the help!
